libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/app/SalesforceHybridSDKManager.java

```diff
@@ -153,4 +179,58 @@ public String getAppType() {
 	public boolean isHybrid() {
 		return true;
 	}
+
+	/**
+	 * Setup global store using config found in assets/www/globalstore.json
+	 */
+	public void setupGlobalStoreFromDefaultConfig() {
```

Comment:
Overriding the method from SmartStoreSDKManager to read from an asset rather than a resource